share/instruments/instruments.xml

```diff
@@ -8843,7 +8843,7 @@
                   <description>Banjo (Tablature)</description>
                   <musicXMLid>pluck.banjo</musicXMLid>
                   <stafftype staffTypePreset="tab5StrCommon">tablature</stafftype>
-                  <!--                  <clef>TAB</clef> -->
+                  <clef>TAB</clef>
```

Comment:
I'm sure there's a good reason, but why did you uncomment all of these?

Reply:
Previously, when you added a change to a tab instrument, it would default to a G clef, which it wouldn't be able to add as it is the wrong staff group. So this reverts them back to tab clefs, which it can then add properly.